popup.html

```diff
@@ -8,17 +8,20 @@
       <div class="Container-text">
         <div class="Container-title">Create a Jira Item</div>
         <div class="Container--italics">Screenshot and URL are provided automatically</div>
-        <div class="Container--italics">All fields are optional and have default values</div>
+        <div class="Container--italics">All fields besides title are optional and have default values</div>
```

Comment:
hm maybe its more straightforward to specify that title is required on its own line?